modules/backend/variables.tf

```diff
@@ -18,4 +18,14 @@ variable "tags" {
   default = {
     terraform = true
   }
+}
+
+variable "s3_access_log_bucket" {
+  type        = string
+  description = "What bucket to write access logs to"
+}
+
+variable "s3_prefix" {
```

Comment:
we should consider a default value or this will break the existing version of the nerves-hub-terraform, probably default it to null.